tests/components/pi_hole/test_init.py

```diff
@@ -29,47 +27,34 @@
         _hole.return_value.data = ZERO_DATA
 
         assert await async_setup_component(
-            hass, pi_hole.DOMAIN, {pi_hole.DOMAIN: [{"host": "pi.hole"}]}
-        )
+            hass, pi_hole.DOMAIN, {pi_hole.DOMAIN: [{
+                "host": "pi.hole"
+            }]})
 
     await hass.async_block_till_done()
 
-    assert (
-        hass.states.get("sensor.pi_hole_ads_blocked_today").name
-        == "Pi-Hole Ads Blocked Today"
-    )
-    assert (
-        hass.states.get("sensor.pi_hole_ads_percentage_blocked_today").name
-        == "Pi-Hole Ads Percentage Blocked Today"
-    )
-    assert (
-        hass.states.get("sensor.pi_hole_dns_queries_cached").name
-        == "Pi-Hole DNS Queries Cached"
-    )
-    assert (
-        hass.states.get("sensor.pi_hole_dns_queries_forwarded").name
-        == "Pi-Hole DNS Queries Forwarded"
-    )
-    assert (
-        hass.states.get("sensor.pi_hole_dns_queries_today").name
-        == "Pi-Hole DNS Queries Today"
-    )
-    assert (
-        hass.states.get("sensor.pi_hole_dns_unique_clients").name
-        == "Pi-Hole DNS Unique Clients"
-    )
-    assert (
-        hass.states.get("sensor.pi_hole_dns_unique_domains").name
-        == "Pi-Hole DNS Unique Domains"
-    )
-    assert (
-        hass.states.get("sensor.pi_hole_domains_blocked").name
-        == "Pi-Hole Domains Blocked"
-    )
-    assert hass.states.get("sensor.pi_hole_seen_clients").name == "Pi-Hole Seen Clients"
+    assert (hass.states.get("sensor.pi_hole_ads_blocked_today").name ==
+            "Pi-Hole Ads Blocked Today")
+    assert (hass.states.get("sensor.pi_hole_ads_percentage_blocked_today").name
+            == "Pi-Hole Ads Percentage Blocked Today")
+    assert (hass.states.get("sensor.pi_hole_dns_queries_cached").name ==
+            "Pi-Hole DNS Queries Cached")
+    assert (hass.states.get("sensor.pi_hole_dns_queries_forwarded").name ==
+            "Pi-Hole DNS Queries Forwarded")
+    assert (hass.states.get("sensor.pi_hole_dns_queries_today").name ==
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4667984)